Makefile

```diff
@@ -119,6 +119,9 @@ kubernetes:
 	bash -f .ci/install_bats.sh
 	bash -f integration/kubernetes/run_kubernetes_tests.sh
 
+ksm:
```

Comment:
does `ksm` also need to go in the test `UNION`?

Reply:
@grahamwhaley no, this test will run in a separate CI that is not part of the PRs